src/components/common/PageHeader/styles.module.css

```diff
@@ -15,8 +15,43 @@
   margin-bottom: var(--space-3);
 }
 
+.border {
+  border-bottom: 1px solid var(--color-border-light);
+}
+
+.pageHeader,
+.actionsWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: var(--space-1);
+}
+
 @media (max-width: 600px) {
   .container {
     padding: var(--space-3) var(--space-2) 0;
   }
+
+  .border {
+    border: 0;
+  }
+
+  .pageHeader {
+    flex-direction: column;
+    align-items: flex-start;
+    gap: var(--space-3);
+  }
+
+  .navWrapper {
+    border-bottom: 1px solid var(--color-border-light);
+    margin-left: -16px;
+    padding-left: 16px;
+    margin-right: -16px;
+    padding-right: 16px;
+    align-self: stretch;
```

Comment:
Is this property necessary?

Reply:
Yes, in order for the element to take up the available width but also stretch further so the border is aligned.